src/App.svelte

```diff
@@ -129,6 +148,7 @@ function isCurrent(person) {
   {#if !isEditing}
     {#if state.shuffled.length > 0}
       <button class="aui-button" on:click={onNextClick}><NextIcon /></button>
+      <button class="aui-button" on:click={onPreviousClick}><PreviousIcon /></button>
```

Comment:
Yeah I was unsure about this. I don't like previous being after next, but we would use next far more than previous so it could be annoying not being the first button... 🤔

Reply:
Have swapped them around now!